c++/src/RLEv2.hh

```diff
@@ -166,6 +166,50 @@ namespace orc {
 
     void next(int16_t* data, uint64_t numValues, const char* notNull) override;
 
+    unsigned char readByte();
+
+    void setBufStart(char* start) {
```

Comment:
The parameter type can be `const char*`. Same for the parameter of `setBufEnd()`

Reply:
Fixed.